### PR TITLE
feat(config): add Zooz Zen77 Z-Wave Ramp Rates

### PR DESCRIPTION
**Pull request by cdnninja**
Adding ramp rate updates for new firmware on zooz 77. Referenced from here: 
https://www.support.getzooz.com/kb/article/550-zen77-s2-dimmer-700-advanced-settings/

### CODE REVIEW

**Comment by cdnninja** [comment]
Ready for review. I am not sure why it isn't liking my titles though.

File: packages/config/config/devices/0x027a/templates/zooz_template.json [hunk 1]
```diff
@@ -215,6 +215,54 @@
 			}
 		]
 	},
+	"dimmer_on_ramp_rate": {
+		"$import": "~/templates/master_template.json#base_0-99_nounit",
+		"label": "Dimmer on ramp rate (fade-in effect for on  operation) when operated manually",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x027a/templates/zooz_template.json]:
```suggestion
		"label": "Ramp Rate On (Manual Control)",
```

File: packages/config/config/devices/0x027a/templates/zooz_template.json [hunk 2]
```diff
@@ -215,6 +215,54 @@
...
+		"options": [
+			{
+				"label": "Instant on",
+				"value": 0
+			}
+		]
+	},
+	"dimmer_off_ramp_rate": {
+		"$import": "~/templates/master_template.json#base_0-99_nounit",
+		"label": "Dimmer off ramp rate (fade-out effect for on  operation) when operated manually",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x027a/templates/zooz_template.json]:
```suggestion
		"label": "Ramp Rate Off (Manual Control)",
```

File: packages/config/config/devices/0x027a/templates/zooz_template.json [hunk 3]
```diff
@@ -215,6 +215,54 @@
...
+		"options": [
+			{
+				"label": "Instant off",
+				"value": 0
+			}
+		]
+	},
+	"zwave_on_dimmer_ramp_rate": {
+		"$import": "~/templates/master_template.json#base_0-99_nounit",
+		"label": "Z-Wave Dimmer on ramp rate (fade-in effect for on zwave operation)",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x027a/templates/zooz_template.json]:
```suggestion
		"label": "Ramp Rate On (Z-Wave)",
```

File: packages/config/config/devices/0x027a/templates/zooz_template.json [hunk 4]
```diff
@@ -215,6 +215,54 @@
...
+		"options": [
+			{
+				"label": "Instant on",
+				"value": 0
+			}
+		]
+	},
+	"zwave_off_dimmer_ramp_rate": {
+		"$import": "~/templates/master_template.json#base_0-99_nounit",
+		"label": "Z-Wave Dimmer off ramp rate (fade-out effect for on zwave operation)",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x027a/templates/zooz_template.json]:
```suggestion
		"label": "Ramp Rate Off (Z-Wave)",
```

File: packages/config/config/devices/0x027a/templates/zooz_template.json [hunk 5]
```diff
@@ -215,6 +215,54 @@
 			}
 		]
 	},
+	"dimmer_on_ramp_rate": {
```
Comment (AlCalzone) [inline packages/config/config/devices/0x027a/templates/zooz_template.json:218]: Please edit the template above this line in a similar fashion

**Review by AlCalzone — CHANGES_REQUESTED** [review]
Looks good, except the labels. Those really don't match the style guide :)
Hmmm, neither does the template you copied this from - we should change that.

**Review by AlCalzone — APPROVED** [review]

**Comment by AlCalzone** [comment]
Thanks!

**Pull request merged**